<launch>
    <node name="gnss_ins_sim_recorder_node" pkg="gnss_ins_sim" type="recorder_node_no_error.py" output="screen">
        <!-- load default params -->
        <rosparam command="load" file="$(find gnss_ins_sim)/config/recorder_no_error.yaml" />

        <!-- configuration -->
    </node>
</launch>